package.json

```diff
@@ -27,8 +27,10 @@
     "start": "node ./node_modules/react-native/local-cli/cli.js start --config ./rn-cli.config.js",
     "run-playground-ios": "cd examples/NavigationPlayground && react-native run-ios",
     "run-playground-android": "cd examples/NavigationPlayground && react-native run-android",
-    "test": "jest",
+    "test": "npm run lint && npm run flow && npm run jest",
+    "jest": "jest",
     "lint": "eslint src",
+    "format": "eslint --fix src",
```

Comment:
There's a few open source plugins to help with this; From the prettier [readme](https://github.com/prettier/prettier#related-projects)

When I integrated one of the wrapping eslint prettier plugins, anything that wasn't prettied correctly would be raised as an issue. That integration also helps nuke the custom `format` script, as it just boils down to `npm run lint -- --fix`